Looped/Model/APIResponse/ReactionTagsResponse.swift

```diff
@@ -1,6 +1,4 @@
-import Foundation
-
-struct ReactionTagsResponse: Codable {
+struct ReactionTagsResponse: Decodable {
     
     let tags: [ReactionTag]
 }
```

Comment:
Generalnie to może być prywatny struct w serwisie który go pobiera bo bo nie korzystasz nigdzie z tego więcej

Reply:
Używam do testowania tej konkretnej metody (2 dodatkowe razy na tę chwilę). Co radzisz w takim wypadku? Czy powinienem zrobić oddzielny model odpowiedzi do testów

Reply:
Build nie przechodził, bo nie sprawdziłem testów - bardzo się posypały - zmieniłem całkiem viewModel, service itp. Pokasowałem testy. Teraz jak jest Rx zrobię od nowa